FreeRTOS-Plus/Source/FreeRTOS-IoT-Libraries-LTS-Beta2/c_sdk/platform/freertos/transport/include/plaintext_freertos.h

```diff
@@ -61,6 +58,16 @@ struct NetworkContext
     Socket_t tcpSocket;
 };
 
+/**
+ * @brief Plain text transport Connect / Disconnect return status.
+ */
+typedef enum PlaintextTransportStatus
+{
+    PLAINTEXT_TRANSPORT_SUCCESS = 0,       /**< Function successfully completed. */
```

Comment:
It is better to avoid a success value as 0. Would be better to start the enum as 1

Reply:
0 is used as a success value in FreeRTOS+TCP sockets and also it is consistent with the MQTT library as well, where 0 is used for indicating the success. I think 0 is  ok in this case as we clearly document the possible return values from the API. What are your thoughts?

Reply:
I have explained the safety reason of avoiding `0` as the success value here: https://github.com/aws/aws-iot-device-sdk-embedded-C/pull/1094/files#r468120026

Reply:
I see the argument and agree with it. I think I will make this change for this PR and we would need to apply the same for all the enums then.